### DISCUSSION TITLE
Hide Route Tabs when using Routing

### DISCUSSION THREAD

**Discussion by maercu** [widgetti/solara#713] [Q&A]
Hi team

Is there a way to hide the "Route-Tabs" when using Routes?

Thanks
Marcel

**Comment by iisakkirotko** [comment]
Hey @maercu!

[The discussion](https://github.com/widgetti/solara/issues/584) has some workarounds for your issue. There is a [PR open](https://github.com/widgetti/solara/pull/657) that should make this possible, but since it is a breaking change, we will release it in Solara 2.0, which should be out in the coming month or two.

**Reply by maercu** [reply]
Thanks for the fast response.
